README.md

```diff
@@ -312,4 +319,64 @@ scripts/convert_umaa_xml.bat
 This repo pulls in a git submodule from [rticonnextdds-cmake-utils](https://github.com/rticommunity/rticonnextdds-cmake-utils) into the /examples/resources folder.  
 
 The `rticonnextdds-cmake-utils` repo provides convenient CMAKE utils to call `rtiddsgen` and pass in idl files as an argument. 
-Use /examples/CMakeLists.txt as a reference for creating a shared library for your UMAA IDL set. 
\ No newline at end of file
+Use /examples/CMakeLists.txt as a reference for creating a shared library for your UMAA IDL set. 
+
+[Back to Use Cases](#use-cases)
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+
+
+## Content Filtered Topics
+
+By using Content Filtered Topics in tandem with XML-Application creation we are able to conveniently  
+filter messages going into a specific data reader using the UMAA Destination ID field.
+
+### Pre-Requisites
+Complete all build/setup as necessary in the [XML App Creation](#xml-app-creation) section. 
+
+### Run
+
+Reference the file umaa_components.xml for the section where the  
+"AnchorCommandReader" is defined. 
+
+You will see the filter defined in XML for a "GUID" value.
+```
+<data_reader name="AnchorCommandReader" topic_ref="UMAA::EO::AnchorControl::AnchorCommand">
+
+                    <!-- Here we are passing the Anchor Controller "GUID" to the Content Filter Topic  
+                        so we only get messages addressed to this GUID  
+                    -->
+                    <content_filter name="src" kind="builtin.sql">
+                        <!-- Can optionally use an ENV variable if desired
+                        <expression> destination = &amp;hex($(ANCHOR_CONTROLLER_GUID)) </expression> -->
+                        <expression>
+                            destination = &amp;hex(05 05 05 05 05 05 05 05 05 05 05 05 05 05 05 05)
+                        </expression>
+                    </content_filter>
+
+                    <datareader_qos base_name="UMAAQOSLibrary::AnchorCommandQOS"/>
+
+                </data_reader>
+```
+Run the Anchor Controller application.  
+
+##### Example:
+```sh
+./build/anchor_controller -f ./resources/umaa_components.xml
+```

```

Comment:
```suggestion
##### Example:

```sh
./build/anchor_controller -f ./resources/umaa_components.xml
```

```

Reply:
This suggestion doesn't look so good, the change was to add a new line character after the title `##### Example`